### ISSUE TITLE
[Bug] Customizing "Your channel" avatar / banner crashes YouTube

### ISSUE THREAD

**Issue by arichorn** [qnblackcat/uYouPlus#125]
### Device info
- iOS/iPadOS version: 15.4.1
- Device model: iPad Pro 10.5 Inch (2017 Model)
- Sideload tool: Altstore
- Version of uYouPlus: 17.15.2-2.1

### Describe the bug
Similar Bug when pressing on notifications it crashes, i also can’t somewhat describe it but I’ll try
Pressing the Channel Picture or Banner during customizing crashes.

### Crash log (if possible)
```

{"app_name":"YouTube","timestamp":"2022-04-24 13:38:45.00 -0500","app_version":"17.15.1","slice_uuid":"c8a6190e-b83b-37f6-9742-00a5ee36940d","build_version":"17.15.1","platform":2,"bundleID":"com.google.ios.youtube.4RRY5SS4JA","share_with_app_devs":0,"is_first_party":0,"bug_type":"309","os_version":"iPhone OS 15.4.1 (19E258)","incident_id":"2D528F11-05E0-44AA-9A34-49CAFEA9BD20","name":"YouTube"}
{
  "uptime" : 190000,
  "procLaunch" : "2022-04-24 13:38:35.4182 -0500",
  "procRole" : "Foreground",
  "version" : 2,
  "userID" : 501,
  "deployVersion" : 210,
  "modelCode" : "iPad7,4",
  "procStartAbsTime" : 4726544024085,
  "coalitionID" : 769,
  "osVersion" : {
    "isEmbedded" : true,
    "train" : "iPhone OS 15.4.1",
    "releaseType" : "User",
    "build" : "19E258"
  },
  "captureTime" : "2022-04-24 13:38:44.6754 -0500",
  "incident" : "2D528F11-05E0-44AA-9A34-49CAFEA9BD20",
  "bug_type" : "309",
  "pid" : 20088,
  "procExitAbsTime" : 4726766031653,
  "cpuType" : "ARM-64",
  "procName" : "YouTube",
  "procPath" : "\/private\/var\/containers\/Bundle\/Application\/03961251-751E-49C3-B089-1CC2FDD5231F\/App.app\/YouTube",
  "bundleInfo" : {"CFBundleShortVersionString":"17.15.1","CFBundleVersion":"17.15.1","CFBundleIdentifier":"com.google.ios.youtube.4RRY5SS4JA","DTAppStoreToolsBuild":"13E500"},
  "storeInfo" : {"deviceIdentifierForVendor":"B2910F7D-04BD-4C02-B072-4B57B9126EE1","thirdParty":true},
  "parentProc" : "launchd",
  "parentPid" : 1,
  "coalitionName" : "com.google.ios.youtube.4RRY5SS4JA",
  "crashReporterKey" : "9ca66166b886aab817dd0b94b39dca18728d47ae",
  "basebandVersion" : "9.01.00",
  "isCorpse" : 1,
  "exception" : {"codes":"0x0000000000000000, 0x0000000000000000","rawCodes":[0,0],"type":"EXC_CRASH","signal":"SIGABRT"},
  "asi" : {"libsystem_c.dylib":["abort() called"]},
  "lastExceptionBacktrace" : [{"imageOffset":599820,"symbol":"__exceptionPreprocess","symbolLocation":216,"imageIndex":6},{"imageOffset":85764,"symbol":"objc_exception_throw","symbolLocation":56,"imageIndex":4},{"imageOffset":1454640,"symbol":"+[NSObject(NSObject) _copyDescription]","symbolLocation":0,"imageIndex":6},{"imageOffset":15393068,"symbol":"-[UIResponder doesNotRecognizeSelector:]","symbolLocation":268,"imageIndex":8},{"imageOffset":180220,"symbol":"___forwarding___","symbolLocation":1472,"imageIndex":6},{"imageOffset":176652,"symbol":"_CF_forwarding_prep_0","symbolLocation":92,"imageIndex":6},{"imageOffset":18724500,"imageIndex":16},{"imageOffset":22178472,"imageIndex":13},{"imageOffset":22166784,"imageIndex":13},{"imageOffset":1363636,"imageIndex":9},{"imageOffset":22166104,"imageIndex":13},{"imageOffset":910348,"imageIndex":9},{"imageOffset":29566144,"imageIndex":13},{"imageOffset":877620,"imageIndex":9},{"imageOffset":917848,"imageIndex":9},{"imageOffset":29566144,"imageIndex":13},{"imageOffset":23885048,"imageIndex":13},{"imageOffset":20714060,"imageIndex":13},{"imageOffset":20713908,"imageIndex":13},{"imageOffset":20713592,"imageIndex":13},{"imageOffset":405652,"symbol":"_dispatch_call_block_and_release","symbolLocation":24,"imageIndex":5},{"imageOffset":409748,"symbol":"_dispatch_client_callout","symbolLocation":16,"imageIndex":5},{"imageOffset":68928,"symbol":"_dispatch_main_queue_drain","symbolLocation":928,"imageIndex":5},{"imageOffset":67984,"symbol":"_dispatch_main_queue_callback_4CF$VARIANT$mp","symbolLocation":36,"imageIndex":5},{"imageOffset":318132,"symbol":"__CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__","symbolLocation":12,"imageIndex":6},{"imageOffset":45016,"symbol":"__CFRunLoopRun","symbolLocation":2544,"imageIndex":6},{"imageOffset":121904,"symbol":"CFRunLoopRunSpecific","symbolLocation":572,"imageIndex":6},{"imageOffset":6536,"symbol":"GSEventRunModal","symbolLocation":160,"imageIndex":7},{"imageOffset":5135440,"symbol":"-[UIApplication _run]","symbolLocation":1080,"imageIndex":8},{"imageOffset":2618320,"symbol":"UIApplicationMain","symbolLocation":336,"imageIndex":8},{"imageOffset":101144,"imageIndex":9},{"imageOffset":99280,"symbol":"start","symbolLocation":444,"imageIndex":10}],
  "faultingThread" : 0,
  "threads" : [{"triggered":true,"id":1019701,"threadState":{"x":[{"value":0},{"value":0},{"value":0},{"value":0},{"value":6888390305},{"value":6165554208},{"value":110},{"value":18446744073709532636},{"value":4320724352,"symbolLocation":0,"symbol":"_main_thread"},{"value":10570892386397256983},{"value":512},{"value":11},{"value":0},{"value":0},{"value":16},{"value":0},{"value":328},{"value":1},{"value":0},{"value":6},{"value":259},{"value":4320724576,"symbolLocation":224,"symbol":"_main_thread"},{"value":0},{"value":0},{"value":50331647},{"value":4320724576,"symbolLocation":224,"symbol":"_main_thread"},{"value":15},{"value":10742373376},{"value":0}],"flavor":"ARM_THREAD_STATE64","lr":{"value":8004008020},"cpsr":{"value":1073741824},"fp":{"value":6165554064},"sp":{"value":6165554032},"esr":{"value":1442840704,"description":" Address size fault"},"pc":{"value":7460961212,"matchesCrashFrame":1},"far":{"value":4554309632}},"queue":"com.apple.main-thread","frames":[{"imageOffset":27580,"symbol":"__pthread_kill","symbolLocation":8,"imageIndex":0},{"imageOffset":55380,"symbol":"pthread_kill","symbolLocation":208,"imageIndex":1},{"imageOffset":128676,"symbol":"abort","symbolLocation":124,"imageIndex":2},{"imageOffset":69080,"symbol":"abort_message","symbolLocation":128,"imageIndex":3},{"imageOffset":5468,"symbol":"demangling_terminate_handler()","symbolLocation":300,"imageIndex":3},{"imageOffset":108776,"symbol":"_objc_terminate()","symbolLocation":124,"imageIndex":4},{"imageOffset":66164,"symbol":"std::__terminate(void (*)())","symbolLocation":16,"imageIndex":3},{"imageOffset":66076,"symbol":"std::terminate()","symbolLocation":60,"imageIndex":3},{"imageOffset":409768,"symbol":"_dispatch_client_callout","symbolLocation":36,"imageIndex":5},{"imageOffset":68928,"symbol":"_dispatch_main_queue_drain","symbolLocation":928,"imageIndex":5},{"imageOffset":67984,"symbol":"_dispatch_main_queue_callback_4CF$VARIANT$mp","symbolLocation":36,"imageIndex":5},{"imageOffset":318132,"symbol":"__CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__","symbolLocation":12,"imageIndex":6},{"imageOffset":45016,"symbol":"__CFRunLoopRun","symbolLocation":2544,"imageIndex":6},{"imageOffset":121904,"symbol":"CFRunLoopRunSpecific","symbolLocation":572,"imageIndex":6},{"imageOffset":6536,"symbol":"GSEventRunModal","symbolLocation":160,"imageIndex":7},{"imageOffset":5135440,"symbol":"-[UIApplication _run]","symbolLocation":1080,"imageIndex":8},{"imageOffset":2618320,"symbol":"UIApplicationMain","symbolLocation":336,"imageIndex":8},{"imageOffset":101144,"imageIndex":9},{"imageOffset":99280,"symbol":"start","symbolLocation":444,"imageIndex":10}]},{"id":1019712,"frames":[{"imageOffset":6452,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":1}]},{"id":1019713,"frames":[{"imageOffset":6452,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":1}]},{"id":1019714,"frames":[{"imageOffset":6452,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":1}]},{"id":1019715,"frames":[{"imageOffset":6452,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":1}]},{"id":1019716,"name":"com.apple.uikit.eventfetch-thread","frames":[{"imageOffset":2732,"symbol":"mach_msg_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":4220,"symbol":"mach_msg","symbolLocation":72,"imageIndex":0},{"imageOffset":26472,"symbol":"__CFRunLoopServiceMachPort","symbolLocation":368,"imageIndex":6},{"imageOffset":43632,"symbol":"__CFRunLoopRun","symbolLocation":1160,"imageIndex":6},{"imageOffset":121904,"symbol":"CFRunLoopRunSpecific","symbolLocation":572,"imageIndex":6},{"imageOffset":97964,"symbol":"-[NSRunLoop(NSRunLoop) runMode:beforeDate:]","symbolLocation":232,"imageIndex":11},{"imageOffset":355984,"symbol":"-[NSRunLoop(NSRunLoop) runUntilDate:]","symbolLocation":88,"imageIndex":11},{"imageOffset":4608160,"symbol":"-[UIEventFetcher threadMain]","symbolLocation":512,"imageIndex":8},{"imageOffset":412972,"symbol":"__NSThread__start__","symbolLocation":792,"imageIndex":11},{"imageOffset":13128,"symbol":"_pthread_start","symbolLocation":116,"imageIndex":1},{"imageOffset":6472,"symbol":"thread_start","symbolLocation":8,"imageIndex":1}]},{"id":1019717,"name":"AVAudioSession Notify Thread","frames":[{"imageOffset":2732,"symbol":"mach_msg_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":4220,"symbol":"mach_msg","symbolLocation":72,"imageIndex":0},{"imageOffset":26472,"symbol":"__CFRunLoopServiceMachPort","symbolLocation":368,"imageIndex":6},{"imageOffset":43632,"symbol":"__CFRunLoopRun","symbolLocation":1160,"imageIndex":6},{"imageOffset":121904,"symbol":"CFRunLoopRunSpecific","symbolLocation":572,"imageIndex":6},{"imageOffset":25720,"symbol":"CADeprecated::GenericRunLoopThread::Entry(void*)","symbolLocation":156,"imageIndex":12},{"imageOffset":63432,"symbol":"CADeprecated::CAPThread::Entry(CADeprecated::CAPThread*)","symbolLocation":88,"imageIndex":12},{"imageOffset":13128,"symbol":"_pthread_start","symbolLocation":116,"imageIndex":1},{"imageOffset":6472,"symbol":"thread_start","symbolLocation":8,"imageIndex":1}]},{"id":1019722,"frames":[{"imageOffset":6452,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":1}]},{"id":1019723,"queue":"com.google.BreakpadQueue","frames":[{"imageOffset":2792,"symbol":"semaphore_wait_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":18776,"symbol":"_dispatch_sema4_wait$VARIANT$mp","symbolLocation":24,"imageIndex":5},{"imageOffset":20392,"symbol":"_dispatch_semaphore_wait_slow","symbolLocation":148,"imageIndex":5},{"imageOffset":32227176,"imageIndex":13},{"imageOffset":32233624,"imageIndex":13},{"imageOffset":32214740,"imageIndex":13},{"imageOffset":32218532,"symbol":"BreakpadUploadNextReportWithParameters","symbolLocation":96,"imageIndex":13},{"imageOffset":32222572,"imageIndex":13},{"imageOffset":405652,"symbol":"_dispatch_call_block_and_release","symbolLocation":24,"imageIndex":5},{"imageOffset":409748,"symbol":"_dispatch_client_callout","symbolLocation":16,"imageIndex":5},{"imageOffset":42808,"symbol":"_dispatch_lane_serial_drain$VARIANT$mp","symbolLocation":644,"imageIndex":5},{"imageOffset":45552,"symbol":"_dispatch_lane_invoke$VARIANT$mp","symbolLocation":408,"imageIndex":5},{"imageOffset":85700,"symbol":"_dispatch_workloop_worker_thread","symbolLocation":632,"imageIndex":5},{"imageOffset":7696,"symbol":"_pthread_wqthread","symbolLocation":284,"imageIndex":1},{"imageOffset":6460,"symbol":"start_wqthread","symbolLocation":8,"imageIndex":1}]},{"id":1019724,"frames":[{"imageOffset":6452,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":1}]},{"id":1019727,"name":"JavaScriptCore libpas scavenger","frames":[{"imageOffset":5252,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":39892,"symbol":"_pthread_cond_wait$VARIANT$mp","symbolLocation":1240,"imageIndex":1},{"imageOffset":17962000,"symbol":"scavenger_thread_main","symbolLocation":960,"imageIndex":14},{"imageOffset":13128,"symbol":"_pthread_start","symbolLocation":116,"imageIndex":1},{"imageOffset":6472,"symbol":"thread_start","symbolLocation":8,"imageIndex":1}]},{"id":1019737,"frames":[{"imageOffset":6452,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":1}]},{"id":1019739,"frames":[{"imageOffset":13064,"symbol":"__bsdthread_terminate","symbolLocation":8,"imageIndex":0},{"imageOffset":19368,"symbol":"_pthread_terminate","symbolLocation":520,"imageIndex":1}]},{"id":1019741,"name":"com.apple.NSURLConnectionLoader","frames":[{"imageOffset":2732,"symbol":"mach_msg_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":4220,"symbol":"mach_msg","symbolLocation":72,"imageIndex":0},{"imageOffset":26472,"symbol":"__CFRunLoopServiceMachPort","symbolLocation":368,"imageIndex":6},{"imageOffset":43632,"symbol":"__CFRunLoopRun","symbolLocation":1160,"imageIndex":6},{"imageOffset":121904,"symbol":"CFRunLoopRunSpecific","symbolLocation":572,"imageIndex":6},{"imageOffset":2391500,"imageIndex":15},{"imageOffset":412972,"symbol":"__NSThread__start__","symbolLocation":792,"imageIndex":11},{"imageOffset":13128,"symbol":"_pthread_start","symbolLocation":116,"imageIndex":1},{"imageOffset":6472,"symbol":"thread_start","symbolLocation":8,"imageIndex":1}]},{"id":1019743,"name":"unnamed","frames":[{"imageOffset":5252,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":39892,"symbol":"_pthread_cond_wait$VARIANT$mp","symbolLocation":1240,"imageIndex":1},{"imageOffset":59138616,"imageIndex":13},{"imageOffset":1768412,"imageIndex":13},{"imageOffset":59140692,"imageIndex":13},{"imageOffset":65577772,"imageIndex":13},{"imageOffset":59141748,"imageIndex":13},{"imageOffset":57246092,"imageIndex":13},{"imageOffset":57385388,"imageIndex":13},{"imageOffset":13128,"symbol":"_pthread_start","symbolLocation":116,"imageIndex":1},{"imageOffset":6472,"symbol":"thread_start","symbolLocation":8,"imageIndex":1}]},{"id":1019744,"frames":[{"imageOffset":6452,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":1}]},{"id":1019747,"frames":[{"imageOffset":6452,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":1}]},{"id":1019765,"name":"com.apple.CFSocket.private","frames":[{"imageOffset":5808,"symbol":"__select","symbolLocation":8,"imageIndex":0},{"imageOffset":668524,"symbol":"__CFSocketManager","symbolLocation":628,"imageIndex":6},{"imageOffset":13128,"symbol":"_pthread_start","symbolLocation":116,"imageIndex":1},{"imageOffset":6472,"symbol":"thread_start","symbolLocation":8,"imageIndex":1}]},{"id":1019781,"name":"ThreadPoolServiceThread","frames":[{"imageOffset":2732,"symbol":"mach_msg_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":4220,"symbol":"mach_msg","symbolLocation":72,"imageIndex":0},{"imageOffset":26472,"symbol":"__CFRunLoopServiceMachPort","symbolLocation":368,"imageIndex":6},{"imageOffset":43632,"symbol":"__CFRunLoopRun","symbolLocation":1160,"imageIndex":6},{"imageOffset":121904,"symbol":"CFRunLoopRunSpecific","symbolLocation":572,"imageIndex":6},{"imageOffset":97964,"symbol":"-[NSRunLoop(NSRunLoop) runMode:beforeDate:]","symbolLocation":232,"imageIndex":11},{"imageOffset":60704676,"imageIndex":13},{"imageOffset":60699640,"imageIndex":13},{"imageOffset":60407552,"imageIndex":13},{"imageOffset":60203760,"imageIndex":13},{"imageOffset":60537856,"imageIndex":13},{"imageOffset":60451024,"imageIndex":13},{"imageOffset":60538244,"imageIndex":13},{"imageOffset":60681844,"imageIndex":13},{"imageOffset":13128,"symbol":"_pthread_start","symbolLocation":116,"imageIndex":1},{"imageOffset":6472,"symbol":"thread_start","symbolLocation":8,"imageIndex":1}]},{"id":1019782,"name":"ThreadPoolForegroundWorker","frames":[{"imageOffset":2732,"symbol":"mach_msg_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":4220,"symbol":"mach_msg","symbolLocation":72,"imageIndex":0},{"imageOffset":60711732,"imageIndex":13},{"imageOffset":60490424,"imageIndex":13},{"imageOffset":60493296,"imageIndex":13},{"imageOffset":60492376,"imageIndex":13},{"imageOffset":60681844,"imageIndex":13},{"imageOffset":13128,"symbol":"_pthread_start","symbolLocation":116,"imageIndex":1},{"imageOffset":6472,"symbol":"thread_start","symbolLocation":8,"imageIndex":1}]},{"id":1019783,"name":"ThreadPoolBackgroundWorker","frames":[{"imageOffset":2732,"symbol":"mach_msg_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":4220,"symbol":"mach_msg","symbolLocation":72,"imageIndex":0},{"imageOffset":60711732,"imageIndex":13},{"imageOffset":60490424,"imageIndex":13},{"imageOffset":60492784,"imageIndex":13},{"imageOffset":60492244,"imageIndex":13},{"imageOffset":60681844,"imageIndex":13},{"imageOffset":13128,"symbol":"_pthread_start","symbolLocation":116,"imageIndex":1},{"imageOffset":6472,"symbol":"thread_start","symbolLocation":8,"imageIndex":1}]},{"id":1019784,"name":"NetworkConfigWatcher","frames":[{"imageOffset":2732,"symbol":"mach_msg_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":4220,"symbol":"mach_msg","symbolLocation":72,"imageIndex":0},{"imageOffset":26472,"symbol":"__CFRunLoopServiceMachPort","symbolLocation":368,"imageIndex":6},{"imageOffset":43632,"symbol":"__CFRunLoopRun","symbolLocation":1160,"imageIndex":6},{"imageOffset":121904,"symbol":"CFRunLoopRunSpecific","symbolLocation":572,"imageIndex":6},{"imageOffset":97964,"symbol":"-[NSRunLoop(NSRunLoop) runMode:beforeDate:]","symbolLocation":232,"imageIndex":11},{"imageOffset":60704676,"imageIndex":13},{"imageOffset":60699640,"imageIndex":13},{"imageOffset":60407552,"imageIndex":13},{"imageOffset":60203760,"imageIndex":13},{"imageOffset":60537856,"imageIndex":13},{"imageOffset":60538244,"imageIndex":13},{"imageOffset":60681844,"imageIndex":13},{"imageOffset":13128,"symbol":"_pthread_start","symbolLocation":116,"imageIndex":1},{"imageOffset":6472,"symbol":"thread_start","symbolLocation":8,"imageIndex":1}]},{"id":1019785,"name":"network","frames":[{"imageOffset":2732,"symbol":"mach_msg_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":4220,"symbol":"mach_msg","symbolLocation":72,"imageIndex":0},{"imageOffset":26472,"symbol":"__CFRunLoopServiceMachPort","symbolLocation":368,"imageIndex":6},{"imageOffset":43632,"symbol":"__CFRunLoopRun","symbolLocation":1160,"imageIndex":6},{"imageOffset":121904,"symbol":"CFRunLoopRunSpecific","symbolLocation":572,"imageIndex":6},{"imageOffset":97964,"symbol":"-[NSRunLoop(NSRunLoop) runMode:beforeDate:]","symbolLocation":232,"imageIndex":11},{"imageOffset":60704676,"imageIndex":13},{"imageOffset":60699640,"imageIndex":13},{"imageOffset":60407552,"imageIndex":13},{"imageOffset":60203760,"imageIndex":13},{"imageOffset":60537856,"imageIndex":13},{"imageOffset":60538244,"imageIndex":13},{"imageOffset":60681844,"imageIndex":13},{"imageOffset":13128,"symbol":"_pthread_start","symbolLocation":116,"imageIndex":1},{"imageOffset":6472,"symbol":"thread_start","symbolLocation":8,"imageIndex":1}]},{"id":1019786,"name":"Network File Thread","frames":[{"imageOffset":2732,"symbol":"mach_msg_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":4220,"symbol":"mach_msg","symbolLocation":72,"imageIndex":0},{"imageOffset":26472,"symbol":"__CFRunLoopServiceMachPort","symbolLocation":368,"imageIndex":6},{"imageOffset":43632,"symbol":"__CFRunLoopRun","symbolLocation":1160,"imageIndex":6},{"imageOffset":121904,"symbol":"CFRunLoopRunSpecific","symbolLocation":572,"imageIndex":6},{"imageOffset":60703748,"imageIndex":13},{"imageOffset":60699640,"imageIndex":13},{"imageOffset":60407552,"imageIndex":13},{"imageOffset":60203760,"imageIndex":13},{"imageOffset":60537856,"imageIndex":13},{"imageOffset":60538244,"imageIndex":13},{"imageOffset":60681844,"imageIndex":13},{"imageOffset":13128,"symbol":"_pthread_start","symbolLocation":116,"imageIndex":1},{"imageOffset":6472,"symbol":"thread_start","symbolLocation":8,"imageIndex":1}]},{"id":1019789,"name":"ThreadPoolForegroundWorker","frames":[{"imageOffset":2732,"symbol":"mach_msg_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":4220,"symbol":"mach_msg","symbolLocation":72,"imageIndex":0},{"imageOffset":60711732,"imageIndex":13},{"imageOffset":60490424,"imageIndex":13},{"imageOffset":60493296,"imageIndex":13},{"imageOffset":60492376,"imageIndex":13},{"imageOffset":60681844,"imageIndex":13},{"imageOffset":13128,"symbol":"_pthread_start","symbolLocation":116,"imageIndex":1},{"imageOffset":6472,"symbol":"thread_start","symbolLocation":8,"imageIndex":1}]}],
  "usedImages" : [
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 7460933632,
    "size" : 212992,
    "uuid" : "3d363709-8d2d-3d1e-8cab-11d232e5d508",
    "path" : "\/usr\/lib\/system\/libsystem_kernel.dylib",
    "name" : "libsystem_kernel.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 8003952640,
    "size" : 69632,
    "uuid" : "7e417a07-6533-3ee7-b0de-623b5fb464fd",
    "path" : "\/usr\/lib\/system\/libsystem_pthread.dylib",
    "name" : "libsystem_pthread.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 6668640256,
    "size" : 507904,
    "uuid" : "0c058d0b-e8f0-36aa-9766-fa72ee100d98",
    "path" : "\/usr\/lib\/system\/libsystem_c.dylib",
    "name" : "libsystem_c.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 6888308736,
    "size" : 94208,
    "uuid" : "7821c7b7-c6b6-3e41-9b53-5c4f754dd470",
    "path" : "\/usr\/lib\/libc++abi.dylib",
    "name" : "libc++abi.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 6887276544,
    "size" : 229376,
    "uuid" : "ce5424bc-ae99-3876-9e37-6202060c145b",
    "path" : "\/usr\/lib\/libobjc.A.dylib",
    "name" : "libobjc.A.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 6490087424,
    "size" : 536576,
    "uuid" : "f14f0161-e0de-3d9c-851e-ad12f95a3073",
    "path" : "\/usr\/lib\/system\/libdispatch.dylib",
    "name" : "libdispatch.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 6493302784,
    "size" : 4444160,
    "uuid" : "a448b016-adb2-3875-8ad5-72e6635def1c",
    "path" : "\/System\/Library\/Frameworks\/CoreFoundation.framework\/CoreFoundation",
    "name" : "CoreFoundation"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 7038910464,
    "size" : 36864,
    "uuid" : "41ace28d-19a4-3850-a2f1-2bd9d6f5b9dd",
    "path" : "\/System\/Library\/PrivateFrameworks\/GraphicsServices.framework\/GraphicsServices",
    "name" : "GraphicsServices"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 6530211840,
    "size" : 24748032,
    "uuid" : "9766f134-0b1c-34a8-a8c9-f8cca8978fde",
    "path" : "\/System\/Library\/PrivateFrameworks\/UIKitCore.framework\/UIKitCore",
    "name" : "UIKitCore"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4301324288,
    "size" : 11026432,
    "uuid" : "c8a6190e-b83b-37f6-9742-00a5ee36940d",
    "path" : "\/private\/var\/containers\/Bundle\/Application\/03961251-751E-49C3-B089-1CC2FDD5231F\/App.app\/YouTube",
    "name" : "YouTube"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4320296960,
    "size" : 344064,
    "uuid" : "5c4972a8-ef81-32dc-a848-42cc7f7874cf",
    "path" : "\/usr\/lib\/dyld",
    "name" : "dyld"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 6517481472,
    "size" : 3031040,
    "uuid" : "7482ad1b-76ee-38b4-8dac-0960f9f9521e",
    "path" : "\/System\/Library\/Frameworks\/Foundation.framework\/Foundation",
    "name" : "Foundation"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 6641803264,
    "size" : 167936,
    "uuid" : "76cba8eb-346c-3ba5-a146-4575c7243cb2",
    "path" : "\/System\/Library\/PrivateFrameworks\/AudioSession.framework\/AudioSession",
    "name" : "AudioSession"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4447322112,
    "size" : 82460672,
    "uuid" : "e9dc8e55-20e8-3733-8644-823bfef5b331",
    "path" : "\/private\/var\/containers\/Bundle\/Application\/03961251-751E-49C3-B089-1CC2FDD5231F\/App.app\/Frameworks\/Module_Framework.framework\/Module_Framework",
    "name" : "Module_Framework"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 6669488128,
    "size" : 19169280,
    "uuid" : "28f95f2f-f2e5-3b11-aee1-3e2bf1126972",
    "path" : "\/System\/Library\/Frameworks\/JavaScriptCore.framework\/JavaScriptCore",
    "name" : "JavaScriptCore"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 6501539840,
    "size" : 4788224,
    "uuid" : "0db401d1-3f6d-349c-a893-ebd36ec7d9a0",
    "path" : "\/System\/Library\/Frameworks\/CFNetwork.framework\/CFNetwork",
    "name" : "CFNetwork"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4360404992,
    "size" : 27361280,
    "uuid" : "b8d1abdb-f036-348f-a8a6-e6e7b9657d74",
    "path" : "\/private\/var\/containers\/Bundle\/Application\/03961251-751E-49C3-B089-1CC2FDD5231F\/App.app\/Frameworks\/uYou.dylib",
    "name" : "uYou.dylib"
  }
],
  "sharedCache" : {
  "base" : 6489767936,
  "size" : 2278064128,
  "uuid" : "f78dcbe3-0a19-3a14-b2c2-88dda63f733f"
},
  "vmSummary" : "ReadOnly portion of Libraries: Total=962.6M resident=0K(0%) swapped_out_or_unallocated=962.6M(100%)\nWritable regions: Total=813.9M written=0K(0%) resident=0K(0%) swapped_out=0K(0%) unallocated=813.9M(100%)\n\n                                VIRTUAL   REGION \nREGION TYPE                        SIZE    COUNT (non-coalesced) \n===========                     =======  ======= \nAccelerate framework               128K        1 \nActivity Tracing                   256K        1 \nCG raster data                    7168K       35 \nColorSync                          320K       20 \nCoreAnimation                      800K       38 \nCoreData                           160K        4 \nCoreData Object IDs               4112K        2 \nCoreUI image data                  656K        6 \nFoundation                         752K        2 \nImage IO                          4896K        9 \nKernel Alloc Once                   32K        1 \nMALLOC                           582.6M      250 \nMALLOC guard page                  192K       12 \nMach message                        16K        1 \nSQLite page cache                 1280K       20 \nSTACK GUARD                       1392K       24 \nStack                             14.6M       24 \nVM_ALLOCATE                        560K       10 \nWebKit Malloc                    192.0M        5 \n__CTF                               759        1 \n__DATA                            64.1M      697 \n__DATA_CONST                      61.8M      704 \n__DATA_DIRTY                      2788K      586 \n__ETC                              112K        1 \n__FONT_DATA                          4K        1 \n__LINKEDIT                       201.1M       17 \n__OBJC_RO                         91.1M        1 \n__OBJC_RW                         3456K        2 \n__TEXT                           761.5M      719 \n__UNICODE                          592K        1 \ndyld private memory               1024K        1 \nlibnetwork                        1664K       24 \nmapped file                      211.8M       42 \nshared memory                       48K        3 \n===========                     =======  ======= \nTOTAL                              2.2G     3265 \n",
  "legacyInfo" : {
  "threadTriggered" : {
    "queue" : "com.apple.main-thread"
  }
},
  "trialInfo" : {
  "rollouts" : [
    {
      "rolloutId" : "5fc94383418129005b4e9ae0",
      "factorPackIds" : {

      },
      "deploymentId" : 240000349
    },
    {
      "rolloutId" : "61030413bfe6dc472e1c980c",
      "factorPackIds" : {

      },
      "deploymentId" : 240000265
    }
  ],
  "experiments" : [
    {
      "treatmentId" : "fb58143a-d0d1-4465-a621-853f75606309",
      "experimentId" : "6192fb082171a2330e561df0",
      "deploymentId" : 400000021
    },
    {
      "treatmentId" : "dc6188db-da76-4777-9ee0-4bd129ce1e4a",
      "experimentId" : "61ae5b540a12026a0805a31f",
      "deploymentId" : 400000018
    }
  ]
}
}
```

**Comment by level3tjg** [comment]
This doesn't seem to be an issue anymore, however it should be fixed by [4b28311](https://github.com/qnblackcat/uYouPlus/commit/4b283110a640ef52608e6f3022937957b5513389) for anyone still affected by this.

**Comment by arichorn** [comment]
YESSSSS! THANKS! 👍
